valhalla/worker.h

```diff
@@ -151,16 +151,8 @@ void ParseApi(const std::string& json_request, Options::Action action, Api& api)
 void ParseApi(const prime_server::http_request_t& http_request, Api& api);
 #endif
 
+std::string jsonify_error(const valhalla_exception_t& exception, const Api& options);
```

Comment:
this is the only real change. just moved the other thing down so the jsonify methods would be closer together